src/server/journal/streamer.h

```diff
@@ -62,7 +63,9 @@ class JournalStreamer {
   bool IsStalled() const;
 
   journal::Journal* journal_;
-  std::vector<uint8_t> pending_buf_;
+
+  size_t pending_buf_mem_size = 0;
+  std::vector<std::string> pending_buf_;
```

Comment:
@chakaz Do you have any ideas why I can not use absl::InlineVector<std::string, 4> here? I've tried but everything got broken

Reply:
I've written the question and understand because of SSO